tests/test_data_module.py

```diff
@@ -107,10 +107,7 @@ def test_idk_targets(data):
     # load idk type dataset - don't pass tokenizer or qa_formatter so we can look
     # directly at output.
     idk_set = EvalQADataset(
-        data,
-        _identity,
-        _identity,
-        loss_type="idk",
+        data, _identity, _identity, loss_type="idk", qualitative_eval=True
```

Comment:
otherwise it returns batch formatted inputs, qualitative evaluation outputs a (question,answer) tuple

Reply:
By passing in the _identity function for the tokenizer and formatter it will just return the raw strings. Meaning we can directly compare them with the file they're sampled from. It just saves us having to decode the dataset output.